Test suite regression

A change introduced in this commit https://github.com/beanstalkd/beanstalkd/commit/4c275d5945299e4562389f9f2ca7c326173d6335 broke the test suite:

```
ct/_ctcheck
....................................................................F...............................

cttest_stats_tube: failure
start server port=38419 pid=284
>72 use tubea
<72 USING tubea
>72 put 0 0 0 1
>72 x
<72 INSERTED 1
>72 delete 1
<72 DELETED
>72 stats-tube tubea
<72 OK 265
<72 ---
name: "tubea"
current-jobs-urgent: 0
current-jobs-ready: 0
current-jobs-reserved: 0
current-jobs-delayed: 0
current-jobs-buried: 0
total-jobs: 1
current-using: 1
current-watching: 0
current-waiting: 0
cmd-delete: 1
cmd-pause-tube: 0
pause: 0
pause-time-left: 0

testserv.c:330: test: strstr(line, sub)
testserv.c:330: "
name: tubea
" not in "---
name: "tubea"
current-jobs-urgent: 0
current-jobs-ready: 0
current-jobs-reserved: 0
current-jobs-delayed: 0
current-jobs-buried: 0
total-jobs: 1
current-using: 1
current-watching: 0
current-waiting: 0
cmd-delete: 1
cmd-pause-tube: 0
pause: 0
pause-time-left: 0

"

1 failures; 0 errors.
make: *** [Makefile:95: check] Error 1
```

It would probably be worthwhile to clean up the extant Travis configuration and set up a new CI pipeline elsewhere, like GH Actions.

--- a/testserv.c
+++ b/testserv.c
@@ -941,7 +941,7 @@ cttest_stats_tube()
 
     mustsend(fd, "stats-tube tubea\r\n");
     ckrespsub(fd, "OK ");
-    ckrespsub(fd, "\nname: tubea\n");
+    ckrespsub(fd, "\nname: \"tubea\"\n");
     mustsend(fd, "stats-tube tubea\r\n");
     ckrespsub(fd, "OK ");
     ckrespsub(fd, "\ncurrent-jobs-urgent: 0\n");
@@ -984,7 +984,7 @@ cttest_stats_tube()
 
     mustsend(fd, "stats-tube default\r\n");
     ckrespsub(fd, "OK ");
-    ckrespsub(fd, "\nname: default\n");
+    ckrespsub(fd, "\nname: \"default\"\n");
     mustsend(fd, "stats-tube default\r\n");
     ckrespsub(fd, "OK ");
     ckrespsub(fd, "\ncurrent-jobs-urgent: 0\n");

